src/extension.ts

```diff
@@ -89,52 +95,30 @@ export async function activate(context: vscode.ExtensionContext) {
         })
         await ext.telemetry.start()
 
-        registerCommand({
-            command: 'aws.login',
-            callback: async () => await ext.awsContextCommands.onCommandLogin(),
-            telemetryName: 'aws_credentialslogin'
-        })
-
-        registerCommand({
-            command: 'aws.credential.profile.create',
-            callback: async () => await ext.awsContextCommands.onCommandCreateCredentialsProfile(),
-            telemetryName: 'aws_credentialscreate'
-        })
+        vscode.commands.registerCommand('aws.login', async () => await ext.awsContextCommands.onCommandLogin())
+        vscode.commands.registerCommand('aws.logout', async () => await ext.awsContextCommands.onCommandLogout())
 
-        registerCommand({
-            command: 'aws.logout',
-            callback: async () => await ext.awsContextCommands.onCommandLogout(),
-            telemetryName: 'aws_credentialslogout'
+        vscode.commands.registerCommand('aws.credential.profile.create', async () => {
+            await ext.awsContextCommands.onCommandCreateCredentialsProfile()
```

Comment:
Metrics calls would be missed if function throws an Error

Reply:
fixed